apps/editors/helpers.py

```diff
@@ -160,6 +175,22 @@ def queue_tabnav(context):
                (ngettext('Moderated Review ({0})', 'Moderated Reviews ({0})',
                          counts['moderated'])
                 .format(counts['moderated'])))]
+    if context.get('unlisted'):
```

Comment:
I'd definitely prefer an if-else here, especially since it would save a needless queue counts query.